.github/workflows/dependabot-automerge.yml

```
@@ -159,11 +164,87 @@ jobs:
         needs.inspect_changed_files.outputs.update_type == 'version-update:semver-minor'
       )
     permissions:
+      administration: read
```

actionlint (1.7.12)

[error] unknown permission scope "administration". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)